src/coreComponents/mesh/generators/VTKUtilities.cpp

```diff
@@ -2058,10 +2064,10 @@ real64 writeNodes( integer const logLevel,
     // TODO: remove this check once the input mesh is cleaned of duplicate points via a filter
     //       and make launch policy parallel again
     GEOS_ERROR_IF( nodeGlobalIds.count( pointGlobalID ) > 0,
-                   GEOS_FMT( "Duplicate point detected: globalID = {}\n"
-                             "Consider cleaning the dataset in Paraview using 'Clean to grid' filter.\n"
-                             "Make sure partitionRefinement is set to 1 or higher (this may help).",
-                             pointGlobalID ) );
+                   GEOS_FMT( "At least one duplicate point detected (globalID = {}).{}\n"
+                             "Potential fixes :\n- Consider cleaning the dataset in Paraview using 'Clean to grid' filter.\n"
```

Comment:
Do we really want to advise the usag of `clean to grid`? My XP with this tool was not great. @jeannepellerin ?

Reply:
I asked her, we don't need to recommend that to the user as it is not reliable. I will remove this advice.